### ISSUE TITLE
No simple example for concurrent connections 

### ISSUE THREAD

**Issue by George3d6** [tokio-rs/tokio#637]
Currently there are no simple examples that show an easy "client side" task such as concurrently "sending" two tcp messages.

Essentially all the trivial examples show all the asynchronous tasks being crammed inside the `for_each` function of a `TcpListener`.

Would you consider adding some examples of the "correct" way of doing a simple tasks such as launching 2 concurrent connections over tcp and waiting until both of them send & receive a message ?

**Comment by George3d6** [comment]
Note, just to give an example of what I'm referring to, something like this:

```
extern crate tokio;
extern crate futures;

use futures::future;
use tokio::{net::TcpStream, prelude::Future};
use std::net::SocketAddr;

fn send(addr: SocketAddr, message: String) {
    let task = TcpStream::connect(&addr)
    .and_then(|stream| tokio::io::write_all(stream, message))
    .map_err(|e| println!("Error: {}", e))
    .map(|_| ());

    tokio::spawn(task);
}

fn main() {
    tokio::run(future::lazy(|| {
        send("8.8.8.8:1234".parse().unwrap(), String::from("46"));
        send("1.1.1.1:4321".parse().unwrap(), String::from("3"));
        Ok(())
    }));
}
```

Though ideally w/o the use of the futures crate explicitly, if possible.

Instead of something more complex, like this: https://github.com/tokio-rs/tokio/blob/master/examples/manual-runtime.rs

**Comment by Earls1** [comment]
I disagree maybe take a step back. I saw a file that had everyone with a cell phone pinged in arizona we got beat to punch. Had value for each person .91-.51. We will not be allowed to create an app that could end kidnapping and home invasions if it worked the cops would b called as soon as line is broken at night and give police lead. Could be used to track people. I go with ajything u want file is arzgps and got took weeks ago.

**Comment by Earls1** [comment]
I agree im so sorry

**Comment by Earls1** [comment]
As long as we,have a place to identify people that should be there and set certain time that if someone is in your yard for 5 secs. At 12 am the app identifys number and calls police from ur phone and their phone.
